word_nerd/decks/tests/test_create_new_deck.py

```diff
@@ -0,0 +1,78 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from decks.models import Deck
+from flashcards.tests.utils_test import create_language_and_add_for_user
+from users.tests.utils_test import register_and_login_user
+
+
+class AddNewLanguage(APITestCase):
```

Comment:
We can also add a test that tries to add a deck with a language pk that is not among user-selected languages